crates/solver-types/src/utils/eip712.rs

```diff
@@ -156,7 +156,7 @@ pub fn reconstruct_permit2_digest(
 	// 2. Compute struct hash for PermitBatchWitnessTransferFrom
 
 	// Type hash for the main struct
-	let permit_type = "PermitBatchWitnessTransferFrom(TokenPermissions[] permitted,address spender,uint256 nonce,uint256 deadline,Permit2Witness witness)MandateOutput(bytes32 oracle,bytes32 settler,uint256 chainId,bytes32 token,uint256 amount,bytes32 recipient,bytes call,bytes context)TokenPermissions(address token,uint256 amount)Permit2Witness(uint32 expires,address inputOracle,MandateOutput[] outputs)";
+	let permit_type = "PermitBatchWitnessTransferFrom(TokenPermissions[] permitted,address spender,uint256 nonce,uint256 deadline,Permit2Witness witness)MandateOutput(bytes32 oracle,bytes32 settler,uint256 chainId,bytes32 token,uint256 amount,bytes32 recipient,bytes callbackData,bytes context)Permit2Witness(uint32 expires,address inputOracle,MandateOutput[] outputs)TokenPermissions(address token,uint256 amount)";
```

Comment:
Thinking about it, maybe we don't need to name it `callbackData` here. I don't see it [here](https://github.com/openintentsframework/oif-contracts/blob/main/src/input/escrow/Permit2WitnessType.sol#L36-L37) either.